config/migrations/20220621162039-insert-internal-x-publication-activities-kanselarij.sparql

```diff
@@ -0,0 +1,41 @@
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX prov: <http://www.w3.org/ns/prov#>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX besluitvorming: <http://data.vlaanderen.be/ns/besluitvorming#>
+
+# Only convert in kanselarij-graph
+
+INSERT {
+    GRAPH <http://mu.semte.ch/graphs/organizations/kanselarij> {
+        ?decisionPub a ext:InternalDecisionPublicationActivity .
+        ?decisionPub mu:uuid ?uuid .
+        ?decisionPub ext:internalDecisionPublicationActivityUsed ?meeting .
+        ?decisionPub prov:startedAtTime ?publicationTime .
+    }
+} WHERE {
+    GRAPH <http://mu.semte.ch/graphs/organizations/kanselarij> {
+        ?meeting a besluit:Vergaderactiviteit .
+        ?meeting ext:releasedDecisions ?publicationTime .
```

Comment:
op productie vallen 4 vergaderingen buiten deze query
```
PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
PREFIX prov: <http://www.w3.org/ns/prov#>
PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
PREFIX besluitvorming: <http://data.vlaanderen.be/ns/besluitvorming#>


select * WHERE {
  GRAPH <http://mu.semte.ch/graphs/organizations/kanselarij> {
    ?meeting a besluit:Vergaderactiviteit .
    ?meeting <http://data.vlaanderen.be/ns/besluit#geplandeStart> ?o .
    FILTER NOT EXISTS { ?meeting ext:releasedDecisions ?publicationTime .}
  }
}
```
2 ervan staan op themis en zijn EP agendas.
De andere 2 staan niet op themis.

Op zich geen probleem denk ik, als de  beslissingen niet zijn vrijgegeven via onze app dan hoeft er ook geen activity te bestaan.
Maar dan moeten we in de frontend ook rekening houden met het feit dat dit dus optioneel is.

Ik bedenk me dan ook:
Met de vooraf aangemaakte modellen, is er dan nog de mogelijkheid om nooit beslissingen/documenten vrij te geven?